flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/PropertiesConverter.java

```diff
@@ -48,7 +61,24 @@ default Map<String, String> toGravitinoCatalogProperties(Configuration flinkConf
    * @return properties for the Flink connector.
    */
   default Map<String, String> toFlinkCatalogProperties(Map<String, String> gravitinoProperties) {
-    return gravitinoProperties;
+    Map<String, String> all = Maps.newHashMap();
+    gravitinoProperties.forEach(
+        (key, value) -> {
+          String flinkConfigKey = key;
+          if (key.startsWith(PropertiesConverter.FLINK_PROPERTY_PREFIX)) {
+            flinkConfigKey = key.substring(PropertiesConverter.FLINK_PROPERTY_PREFIX.length());
+          }
+          all.put(flinkConfigKey, value);
+        });
+    Map<String, String> allProperties = transformPropertiesToFlinkCatalog(all);
+    allProperties.put(CommonCatalogOptions.CATALOG_TYPE.key(), getFlinkCatalogType());
+    return allProperties;
+  }
+
+  String transformPropertiesToGravitinoCatalog(String configKey);
```

Comment:
Could you add Java doc for new interfaces? and please polish existing Java doc for `toFlinkCatalogProperties` and `toGravitinoCatalogProperties`

Reply:
fixed